src/registrar/assets/js/get-gov-admin.js

```diff
@@ -750,3 +760,113 @@ function initializeWidgetOnList(list, parentId) {
         });
     }
 })();
+
+
+/** An IIFE for copy summary button (appears in DomainRegistry models)
+*/
+(function dynamicPortfolioFields(){
+
+    document.addEventListener('DOMContentLoaded', function() {
+        
+        let isPortfolioPage = document.querySelector("#portfolio_form");
+        if (!isPortfolioPage) {
+            return;
+        }
+        
+        // $ symbolically denotes that this is using jQuery
+        let $federalAgency = django.jQuery("#id_federal_agency");
+        let organizationType = document.querySelector("#id_organization_type");
```

Comment:
nitpick: getElementById is faster (https://stackoverflow.com/questions/57159219/performance-of-getelementbyid-vs-getelementsbyclassname-vs-queryselector)